pkg/debugapi/welcome_message.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package debugapi
+
+import (
+	"io/ioutil"
+	"net/http"
+
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+)
+
+type welcomeMessageResponse struct {
+	WelcomeMesssage string `json:"welcome_message"`
+}
+
+func (s *server) welcomeMessageSyncedHandler(w http.ResponseWriter, r *http.Request) {
+	val := s.P2P.WelcomeMessageSynced()
+	jsonhttp.OK(w, welcomeMessageResponse{
+		WelcomeMesssage: val,
+	})
+}
+
+func (s *server) setWelcomeMessageHandler(w http.ResponseWriter, r *http.Request) {
+	data, err := ioutil.ReadAll(r.Body)
```

Comment:
if i generate a very large random string (gigabytes order) and upload it to this endpoint it will make the node crumble and die. this should be limited to a predefined size and used with `Read`

Reply:
@acud - excellent point. I'll look to sanitizing all inputs in the PR.